src/CentreonLegacy/Core/Utils/Utils.php

```diff
@@ -212,14 +212,11 @@ public function encodePass($password, $algo = 'md5')
             case 'md5':
                 $encodePassword .= 'md5__' . md5($password);
                 break;
-            case 'sha1':
-                $encodePassword .= 'sha1__' . sha1($password);
-                break;
             case PASSWORD_BCRYPT:
                 $encodePassword = password_hash($password, PASSWORD_BCRYPT);
                 break;
             default:
-                $encodePassword .= 'md5__' . md5($password);
+                $encodePassword = password_hash($password, PASSWORD_BCRYPT);
```

Comment:
I suggest you delete this line and move the 'default' to just above the "PASSWORD_BCRYPT:" line.